### ISSUE TITLE
401 Authorized

### ISSUE THREAD

**Issue by FunixBoss** [Gimenz/insta-fetcher#34]
I think I got my ip banned from accessing the instagram api too much. But I changed the proxy and cookies (new account) I still can't access it

**Comment by andreineamtu** [comment]
Yes I get that error too after a few requests. Is there a fix for this?
